bokehjs/test/integration/regressions.ts

```diff
@@ -25,7 +25,7 @@ describe("Bug", () => {
         y_range: new DataRange1d(),
       })
       const source = new ColumnDataSource({data: {x: [["a", "b"], ["b", "c"]], y: [1, 2]}})
-      p.vbar({x: {field: "x"}, top: {field: "y"}, source})
+      p.vbar({x: {field: "x"}, top: {field: "y"}, width: 0.1, source})
```

Comment:
what is this addition for?

Reply:
The test originally depended on `null` to 0 conversion, and produced zero width bars, which due to non-zero line width appeared to have non-zero width. The more I test bokehjs alone, the more cases like this get revealed.

Reply:
I see, I thought there was supposed to be a default width of 1 for bars?

Reply:
There is a default, but a single fat bar (of width 1) doesn't seem appropriate for this test. It looks like it's hiding something underneath the bar.